packages/preview/fontawesome/0.2.1/README.md

```diff
@@ -0,0 +1,125 @@
+# typst-fontawesome
+
+A Typst library for Font Awesome icons through the desktop fonts.
+
+p.s. The library is based on the Font Awesome 6 desktop fonts (v6.5.2)
+
+## Usage
+
+### Install the fonts
+
+You can download the fonts from the official website: https://fontawesome.com/download
+
+Or you can use the helper script to download the fonts and metadata:
+
+`python helper.py -dd -v {version}`
+
+Here `-dd` means to download and extract the zip file. You can use `-d` to only download the zip file.
+
+After downloading the zip file, you can install the fonts depending on your OS.
+
+#### Typst web app
+
+You can simply upload the `otf` files to the web app and use them with this package.
+
+#### Mac
+
+You can double click the `otf` files to install them.
+
+#### Windows
+
+You can right-click the `otf` files and select `Install`.
+
+### Import the library
+
+#### Using the typst packages
+
+You can install the library using the typst packages:
+
+`#import "@preview/fontawesome:0.2.0": *`
```

Comment:
You probably want to bump the version in this snippet too.

Reply:
Fixed